src/simulators/matrix_product_state/svd.cpp

```diff
@@ -136,6 +137,27 @@ void reduce_zeros(cmatrix_t &U, rvector_t &S, cmatrix_t &V,
   }
 }
 
+void validate_SVD_result(cmatrix_t &A, cmatrix_t &U,rvector_t &S,cmatrix_t &V) {
+  const uint_t nrows = A.GetRows(), ncols = A.GetColumns();
+  cmatrix_t diag_S = diag(S, nrows, ncols);
+  cmatrix_t product = U*diag_S;
+  product = product * AER::Utils::dagger(V);
+  std::cout << "matrices after SVD:" <<std::endl;
```

Comment:
All these debug print statements should be removed

Reply:
removed the debug prints.